netlify.toml

```diff
@@ -680,3 +680,8 @@
 [[redirects]]
     from = "/docs/integrate/client/js/index"
     to = "/docs/integrate/client/js"
+
+# Added: 2021-08-10
```

Comment:
Not sure why there are redirects appearing in this PR?

Reply:
I think the redirects were updated after this PR was posted, but before it was published.

Reply:
Looks like this is a redirect case from a recent change to support default files in directories that `safe_redirects.py` doesn't handle. I'll push a fix.